.travis.yml

```diff
@@ -1,12 +1,15 @@
 sudo: false
 language: ruby
 before_install:
-  - gem install bundler -v 1.14.6
+  - gem install bundler -v 1.16.1
   - gem install rubocop
```

Comment:
Small suggestion: you can include the `rubocop` gem in the Gemfile under the group `:test`, and then you won't have to specify it here.